### PR TITLE
[FIX] Translation interpolations for many languages

### LINKED ISSUE #12001: [BUG] interface in Russian translation Prune: Prune_Warning_
### Steps to reproduce:

1.  Go to 'Prune Messages' 

### Expected behavior:

Here's how the message about deleting messages in the English interface looks.
prune1

### Actual behavior:

Here's how the message about deleting messages in the Russian interface looks.
He is not!
prune2
### Server Setup Information:

- Version of Rocket.Chat Server: 0.69.0
- Operating System: Ubuntu 16.04.5 LTS
- Deployment Method: snap
- Number of Running Instances: 1
- DB Replicaset Oplog: Enabled
- NodeJS Version: v8.11.3

### Additional context

Also there is no processing of keys in the Russian translation:
 "Prune_Warning_before": "This will delete all %s in %s before %s.",
 "Prune_Warning_after": "This will delete all %s in %s after %s.",
 "Prune_Warning_between": "This will delete all %s in %s between %s and %s.",

### LINKED ISSUE #12130: Prune messages: warning field is empty on non EN localization
### Description:

When pruning messages the warning field is empty on all non EN localizations.

related to  #12001

### Steps to reproduce:

1. change on profile localization from EN eg. to DE or FR, log out, log in again
2. go to a channel/room and select "prune messages"

### Expected behavior:

the warning field should contain the warning text.

prune_en2

### Actual behavior:

warning field is empty:

prune_fr

prune_german

### Server Setup Information:

- Version of Rocket.Chat Server: 0.69.2
- Operating System: Ubuntu 16.04 LTS
- Deployment Method: bundle self build
- Number of Running Instances: 1
- DB Replicaset Oplog: yes
- NodeJS Version: v8.11.3
- MongoDB Version: 3.4.17

### PR DESCRIPTION
**Pull request by fliptrail**
Closes #12130, Closes #12001

Fixes these and some string modifier issues.

### Initial State
imported

### Final State
Screenshot from 2019-03-17 00-22-40

### CODE REVIEW

**Comment by engelgabriel** [comment]
@sampaiodiego do we need to upload the files directly to LingoHub once this is merged? Or are we still doing the auto-fetch from here

**Comment by sampaiodiego** [comment]
@engelgabriel it is currently turned off for languages other than english.. I'll turned it on again as we're already missing some translations there.

**Comment by fliptrail** [comment]
@sampaiodiego Please review now.

File: packages/rocketchat-i18n/i18n/ru.i18n.json [hunk 1]
```diff
@@ -2191,10 +2191,10 @@
   "Prune_finished": "Очистка завершена",
   "Prune_Messages": "Черновики",
   "Prune_Modal": "Вы уверены, что хотите удалить эти сообщения? Удаленные сообщения не могут быть восстановлены.",
-  "Prune_Warning_all": "Это удалит все %s в %s!",
-  "Prune_Warning_before": "Это приведет к удалению всех %s в %s до %s.",
-  "Prune_Warning_after": "Это приведет к удалению всех %s в %s после %s.",
-  "Prune_Warning_between": "Это приведет к удалению всех %s в %s между %s и %s.",
+  " %srning_all": "Это удалит все %s в %s!",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/ru.i18n.json]:
```suggestion
  "Prune_Warning_all": "Это удалит все %s в %s!",
```

File: packages/rocketchat-i18n/i18n/ru.i18n.json [hunk 2]
```diff
@@ -2191,10 +2191,10 @@
...
+  " %srning_before": "Это приведет к удалению всех %s в %s до %s.",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/ru.i18n.json]:
```suggestion
  "Prune_Warning_before": "Это приведет к удалению всех %s в %s до %s.",
```

File: packages/rocketchat-i18n/i18n/ru.i18n.json [hunk 3]
```diff
@@ -2191,10 +2191,10 @@
...
+  " %srning_after": "Это приведет к удалению всех %s в %s после %s.",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/ru.i18n.json]:
```suggestion
  "Prune_Warning_after": "Это приведет к удалению всех %s в %s после %s.",
```

File: packages/rocketchat-i18n/i18n/ru.i18n.json [hunk 4]
```diff
@@ -2191,10 +2191,10 @@
...
+  " %srning_between": "Это приведет к удалению всех %s в %s между %s и %s.",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/ru.i18n.json]:
```suggestion
  "Prune_Warning_between": "Это приведет к удалению всех %s в %s между %s и %s.",
```

File: packages/rocketchat-i18n/i18n/sk-SK.i18n.json [hunk 5]
```diff
@@ -2028,10 +2028,10 @@
   "Prune_finished": "Prerezávanie bolo dokončené",
   "Prune_Messages": "Prerežte Správy",
   "Prune_Modal": "Naozaj chcete tieto správy prerezávať? Prerezané správy nie je možné obnoviť.",
-  "Prune_Warning_all": "Tým sa odstránia všetky% s v% s!",
-  "Prune_Warning_before": "Tým sa odstráni všetky% s v% s pred% s.",
-  "Prune_Warning_after": "Tým sa odstráni všetky% s v% s po% s.",
-  "Prune_Warning_between": "Tým sa odstránia všetky% s v% s medzi% s a% s.",
+  " %srning_all": "Tým sa odstránia všetky% s v% s!",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/sk-SK.i18n.json]:
```suggestion
  "Prune_Warning_all": "Tým sa odstránia všetky %s v %s!",
```

File: packages/rocketchat-i18n/i18n/sk-SK.i18n.json [hunk 6]
```diff
@@ -2028,10 +2028,10 @@
...
+  " %srning_before": "Tým sa odstráni všetky% s v% s pred% s.",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/sk-SK.i18n.json]:
```suggestion
  "Prune_Warning_before": "Tým sa odstráni všetky %s v %s pred %s.",
```

File: packages/rocketchat-i18n/i18n/sk-SK.i18n.json [hunk 7]
```diff
@@ -2028,10 +2028,10 @@
...
+  " %srning_after": "Tým sa odstráni všetky% s v% s po% s.",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/sk-SK.i18n.json]:
```suggestion
  "Prune_Warning_after": "Tým sa odstráni všetky %s v %s po %s.",
```

File: packages/rocketchat-i18n/i18n/sk-SK.i18n.json [hunk 8]
```diff
@@ -2028,10 +2028,10 @@
...
+  " %srning_between": "Tým sa odstránia všetky% s v% s medzi% s a% s.",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/sk-SK.i18n.json]:
```suggestion
  "Prune_Warning_between": "Tým sa odstránia všetky %s v %s medzi %s a %s.",
```

File: packages/rocketchat-i18n/i18n/cy.i18n.json [hunk 9]
```diff
@@ -815,8 +815,8 @@
   "Create_new": "Creu newydd",
   "Create_unique_rules_for_this_channel": "Creu rheolau unigryw ar gyfer y sianel hon",
   "Created_at": "Wedi'i greu yn",
-  "Created_at_s_by_s": "Crëwyd yn <strong>% s</strong>gan <strong>% s</strong>",
-  "Created_at_s_by_s_triggered_by_s": "Crëwyd yn <strong>% s</strong>gan <strong>% s</strong>a achoswyd gan <strong>% s</strong>",
+  "Created_at_s_by_s": "Crëwyd yn <strong> %s</strong>gan <strong> %s</strong>",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/cy.i18n.json]:
```suggestion
  "Created_at_s_by_s": "Crëwyd yn <strong>%s</strong> gan <strong>%s</strong>",
```

File: packages/rocketchat-i18n/i18n/cy.i18n.json [hunk 10]
```diff
@@ -940,10 +940,10 @@
...
   "Dry_run": "Rhedeg sych",
   "Dry_run_description": "Dim ond un e-bost fydd yn ei anfon, i'r un cyfeiriad ag O. Rhaid i'r e-bost fod yn ddefnyddiwr dilys.",
-  "Duplicate_archived_channel_name": "Mae Sianel archif gydag enw `#% s` yn bodoli",
-  "Duplicate_archived_private_group_name": "Mae Grŵp Preifat archif gydag enw '% s' yn bodoli",
-  "Duplicate_channel_name": "Mae Sianel gydag enw '% s' yn bodoli",
-  "Duplicate_private_group_name": "Mae Grŵp Preifat gyda'r enw '% s' yn bodoli",
+  "Duplicate_archived_channel_name": "Mae Sianel archif gydag enw `# %s` yn bodoli",
+  "Duplicate_archived_private_group_name": "Mae Grŵp Preifat archif gydag enw ' %s' yn bodoli",
+  "Duplicate_channel_name": "Mae Sianel gydag enw ' %s' yn bodoli",
+  "Duplicate_private_group_name": "Mae Grŵp Preifat gyda'r enw ' %s' yn bodoli",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/cy.i18n.json]:
```suggestion
  "Duplicate_private_group_name": "Mae Grŵp Preifat gyda'r enw '%s' yn bodoli",
```

File: packages/rocketchat-i18n/i18n/cy.i18n.json [hunk 11]
```diff
@@ -1111,7 +1111,7 @@
   "every_six_hours": "Unwaith bob chwe awr",
   "every_day": "Unwaith bob dydd",
   "Everyone_can_access_this_channel": "Gall pawb gael mynediad i'r sianel hon",
-  "Example_s": "Enghraifft: <code class = \"inline\" >% s</code>",
+  "Example_s": "Enghraifft: <code class = \"inline\" > %s</code>",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/cy.i18n.json]:
```suggestion
  "Example_s": "Enghraifft: <code class = \"inline\" >%s</code>",
```

File: packages/rocketchat-i18n/i18n/cy.i18n.json [hunk 12]
```diff
@@ -1265,12 +1265,12 @@
   "Hide_Avatars": "Cuddio Avatars",
   "Hide_counter": "Cuddio cownter",
   "Hide_flextab": "Cuddio Bar Barhau Cywir gyda Chliciwch",
-  "Hide_Group_Warning": "Ydych chi'n siŵr eich bod am guddio'r grŵp \"% s\"?",
-  "Hide_Livechat_Warning": "Ydych chi'n siŵr eich bod am guddio'r bywladwr gyda \"% s\"?",
-  "Hide_Private_Warning": "Ydych chi'n siŵr eich bod am guddio'r drafodaeth gyda \"% s\"?",
+  "Hide_Group_Warning": "Ydych chi'n siŵr eich bod am guddio'r grŵp \" %s\"?",
+  "Hide_Livechat_Warning": "Ydych chi'n siŵr eich bod am guddio'r bywladwr gyda \" %s\"?",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/cy.i18n.json]:
```suggestion
  "Hide_Livechat_Warning": "Ydych chi'n siŵr eich bod am guddio'r bywladwr gyda \"%s\"?",
```

**Comment by fliptrail** [comment]
Looks like there was a problem with my RegEx. Will see to it tomorrow.

**Pull request merged**